spec/models/shopify_import/custom_collection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::CustomCollection, type: :model do
+  subject { described_class }
+  let(:credentials) do
+    { api_key: 'api_key', password: 'password', shop_domain: 'shop_domain.myshopify.com' }.freeze
+  end
+
+  before { authenticate_with_shopify(credentials) }
```

Comment:
You can use `before { authenticate_with_shopify }`. I'll send you credentials for the test store so you can create collections that you need for the specs.